RFC: problem with DW_OP_GNU_deref_type and dwarf's get_base_type callback
http://sourceware.org/ml/gdb-patches/2012-03/msg00166.html
http://sourceware.org/ml/gdb-cvs/2012-03/msg00118.html

### src/gdb/ChangeLog	2012/03/06 22:48:49	1.13968
### src/gdb/ChangeLog	2012/03/06 23:41:47	1.13969
## -1,3 +1,8 @@
+2012-03-06  Joel Brobecker  <brobecker@adacore.com>
+
+	* dwarf2read.c (dwarf2_get_die_type): Pass absolute offset
+	in call to get_die_type_at_offset.
+
 2012-03-06  Stan Shebs  <stan@codesourcery.com>
 
 	* mi/mi-cmd-break.c: Enforce coding standards, fix comments.
--- src/gdb/dwarf2read.c	2012/03/02 01:55:15	1.618
+++ src/gdb/dwarf2read.c	2012/03/06 23:41:50	1.619
@@ -14269,7 +14269,7 @@
 		     struct dwarf2_per_cu_data *per_cu)
 {
   dw2_setup (per_cu->objfile);
-  return get_die_type_at_offset (die_offset, per_cu);
+  return get_die_type_at_offset (per_cu->offset + die_offset, per_cu);
 }
 
 /* Follow the signature attribute ATTR in SRC_DIE.
